state_of_css.yml

```
@@ -456,14 +654,14 @@ translations:
     t: :focus-visible
 
   - key: features_others.form_controls
-    t: Управління полями форми
+    t: Елементи керування формою
```

```suggestion
    t: Формові елементи
```
Краще спростити отак

---

ні, поки будуть кругом саме елементи керування (controls), бо є формові елементи, які не керують